### PR TITLE
Hungarian language update #15

### PR DESCRIPTION
**Pull request by big4billy**
Full fuzzy removals + new additions on artifacts' texts.

### CODE REVIEW

File: files/lang/hu.po [hunk 1]
```diff
@@ -676,15 +670,15 @@ msgstr ""
...
 msgid ""
 "Surrendering costs gold. However if you pay the ransom, the hero and all of "
 "his or her surviving creatures will be available to recruit again. The cost "
 "of surrender is half of the total cost of the non-temporary troops remaining "
 "in the army."
 msgstr ""
 "A megadás aranyba kerül. Ha megfizeted a váltságdíjat, akkor a hőst és az "
-"összes túlélő lényét újra toborozhatod."
+"összes túlélő lényét újra toborozhatod. A megadás ára a csapatban maradó "
+"egységek árának fele."
```
[the remark this answers is not in the document]
Comment (big4billy) [reply files/lang/hu.po]: I addad some further explanation in brackets.

File: files/lang/hu.po [hunk 2]
```diff
@@ -7623,7 +7602,6 @@ msgid ""
 "troops in the army."
 msgstr "A(z) %{name} %{count} százalékkal csökkenti a megadás költségét."
```
[the remark this answers is not in the document]
Comment (big4billy) [reply files/lang/hu.po]: Corrected

File: files/lang/hu.po [hunk 3]
```diff
@@ -9787,11 +9799,12 @@ msgstr ""
...
 msgstr "Államférfi pennája"
 
-#, fuzzy
 msgid ""
 "The %{name} reduces the cost of surrender to %{count} percent of the total "
 "cost of troops the hero has in their army."
-msgstr "A(z) %{name} %{count} százalékkal csökkenti a megadás költségét."
+msgstr ""
+"Az %{name} a hős seregét alkotó egységek összköltségének %{count} "
+"százalékával csökkenti a megadás költségét."
```
[the remark this answers is not in the document]
Comment (big4billy) [reply files/lang/hu.po]: Corrected

**Review by ihhub — APPROVED** [review]

**Comment by ihhub** [comment]
@big4billy , huge thanks for the update!

**Pull request merged**